src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -424,7 +424,8 @@ void EnElf_Init(Actor* thisx, PlayState* play) {
     }
 }
 
-void func_80A0299C(EnElf* this, s32 arg1) {
+//TODO: Technically a noop, but the argument and values passed to it suggest it may have done *something*.
```

Comment:
Nit: add space between `//` and `TODO`.

Although, maybe this function is related to `func_80A029A8` below? In that case I think it's probably to name these two functions together somehow (though in another PR probably)